logic/src/commonMain/kotlin/com/wire/kalium/logic/data/prekey/PreKey.kt

```diff
@@ -0,0 +1,6 @@
+package com.wire.kalium.logic.data.prekey
+
+data class PreKey(
```

Comment:
Why do we need a `PreKey` type in the logic module? Do we want expose prekeys in the public interface? Otherwise wouldn't it be enough with a PreKey type in cryptography and in the network module?

Reply:
@typfel yes it could be enough..But I wanted that the Crypto module stay independent  from any other module except the logic

Reply:
i also had the same suggestion since there is only 2 ways preyKeys are needed 
1- get some from BE and forword to crypto
2- generate some in crypto and send to the network
we can keep both seperated by having 2 models `PreKeyCrypto` and `PreKeyDTO` and a mapper in `logic`
in this case nor crypto or network will be aware of the other